client/app/components/form/MultiStepButtons.tsx

```diff
@@ -54,33 +51,25 @@ const SubmitButton: React.FunctionComponent<SubmitButtonProps> = ({
               Back
             </Button>
           ))}
-        {!isFinalStep && !submitEveryStep && (
-          <Link href={`${baseUrl}/${step + 2}`}>
-            <Button variant="contained" type="button" disabled={isFinalStep}>
-              Next
-            </Button>
-          </Link>
-        )}
-        {isFinalStep && !submitEveryStep && (
+        {/* When the form is editable, the form should be submitted when navigating between steps */}
```

Comment:
I removed the `submitEveryStep` prop because I can't think of a case where we'd want to not submit. If we didn't submit the first step then the operation wouldn't be created, and submitting the second step would break.

Reply:
Makes sense 👍  In the future if we need a reason to prevent form submission I think we could add something like a `disableSubmit` prop.